src/common/components/signUpModal/signUpModal.component.spec.js

```diff
@@ -344,13 +345,13 @@ describe('signUpForm', function () {
       it('should use saved data from $scope', () => {
         $ctrl.$scope.countryCode = 'UK';
         $ctrl.$scope.streetAddress = user.streetAddress;
-        $ctrl.$scope.streetAddressExtended = user.streetAddressExtended;
+        $ctrl.$scope.streetAddressExtended = '';
         $ctrl.$scope.internationalAddressLine3 = 'internationalAddressLine3';
         $ctrl.$scope.internationalAddressLine4 = 'internationalAddressLine4';
         $ctrl.$scope.city = user.city;
         $ctrl.$scope.state = user.state;
         $ctrl.$scope.zipCode = user.zipCode;
-        $ctrl.$scope.primaryPhone = user.primaryPhone;
+        $ctrl.$scope.primaryPhone = '';
```

Comment:
Why are we using empty strings instead of the extended address and phone number?

Reply:
Thank you for catching that. I've fixed it now